src/containers/edit-profile/profile-tab/profile-tabs/profile-general-tab/ProfileGeneralTab.tsx

```diff
@@ -181,9 +252,15 @@ const ProfileGeneralTab: FC<ProfileGeneralTabProps> = ({ user }) => {
               >
                 https://
               </InputAdornment>
-            )
+            ),
+            sx: styles.videoLinkInput
           }}
+          errorMsg={t(errors.videoLink)}
           fullWidth
+          onBlur={handleBlur('videoLink')}
+          onChange={handleInputChange('videoLink')}
+          placeholder='youtube.com/my-video'
+          value={data.videoLink?.replace('https://', '')}
```

Comment:
I think we should provide default value if our backend will not send us data.videoLink to increase safety. Example:
```suggestion
          value={data.videoLink?.replace('https://', '') ?? ""}
```

Reply:
After examining this line, I realized that the optional chaining operator is redundant here, since I'm already checking the value in [line 48](https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1795/files#diff-a6ecf36f69ddeb793427e9d657db2d5934f6fe82a40bf5381432c8f87a63cc60R48).
So I simply removed the optional chaining operator.